docs/attribution/0002-firefox-desktop.rst

```diff
@@ -8,6 +8,8 @@
 Firefox desktop attribution
 ===========================
 
+TODO: add something about `data-download-version` being checked to apened stub attribution.
```

Comment:
I know this needs to be done, I'd like to get the other changes merged before it gets too stale if it's okay to merge a todo.

Reply:
So should we leave this for another PR?

Reply:
Yes please, this one keeps getting more and more out of date and I don't want to hold it up any longer.

Reply:
Nevermind. I fixed it :)